From 776c8fb64c4195e86e1b4c77606eab97ff37a741 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Wed, 17 Jun 2020 07:37:38 -0500
Subject: [PATCH 5/6] configs: r8a774a1_beacon-rzg2m: Enable CMD_PART

The newly upgraded scripts require CMD_PART to be built into U-Boot.
This patch adds the function so the finduuid script can operate
properly.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 configs/r8a774a1_beacon-rzg2m_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/r8a774a1_beacon-rzg2m_defconfig b/configs/r8a774a1_beacon-rzg2m_defconfig
index 3b7995cd44..9ec037e382 100644
--- a/configs/r8a774a1_beacon-rzg2m_defconfig
+++ b/configs/r8a774a1_beacon-rzg2m_defconfig
@@ -18,6 +18,7 @@ CONFIG_CMD_BOOTZ=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
+CONFIG_CMD_PART=y
 CONFIG_CMD_USB=y
 CONFIG_CMD_DHCP=y
 CONFIG_CMD_MII=y
@@ -64,4 +65,3 @@ CONFIG_USB_EHCI_GENERIC=y
 CONFIG_USB_STORAGE=y
 CONFIG_OF_LIBFDT_OVERLAY=y
 CONFIG_SMBIOS_MANUFACTURER=""
-CONFIG_PRAM=1
-- 
2.17.1

